src/libYARP_math/src/Quaternion.cpp

```diff
@@ -17,6 +17,11 @@ class QuaternionPortContentHeader
 public:
     yarp::os::NetInt32 listTag;
     yarp::os::NetInt32 listLen;
+    QuaternionPortContentHeader()
+    {
+        listTag = 0;
+        listLen = 0;
+    }
```

Comment:
@Nicogene What is the purpose of this? 
Can we use "QuaternionPortContentHeader() = default" instead? Or at least you should use the initialization list...

Reply:
The purpose is to initialise `listTag` and `listLen`, yes we can use the initialization list.